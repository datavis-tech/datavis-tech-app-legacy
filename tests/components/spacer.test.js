import React from 'react'
import { shallow } from 'enzyme'

import Spacer from '../../components/spacer'

describe('spacer', () => {
  let sut

  beforeEach(() => {
    sut = shallow(<Spacer />)
  })

  it(`should have default style 'paddingTop:50px'`, () => {
    expect(sut.find('div').prop('style').paddingTop).toBe('50px')
  })

  describe('if space was provided', () => {
    let space = '100px'

    beforeEach(() => {
      sut.setProps({space})
    })

    it(`should have overriden style 'paddingTop:${space}'`, () => {
      expect(sut.find('div').prop('style').paddingTop).toBe(space)
    })

  })

})
