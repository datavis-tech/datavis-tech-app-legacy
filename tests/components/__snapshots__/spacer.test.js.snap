// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`spacer if space was provided should have overriden style 'paddingTop:100px' 1`] = `
<div
  style={
    Object {
      "paddingTop": "100px",
    }
  }
/>
`;

exports[`spacer should have default style 'paddingTop:50px' 1`] = `
<div
  style={
    Object {
      "paddingTop": "50px",
    }
  }
/>
`;
